src/test/common/wiremock.js

```diff
@@ -52,10 +55,40 @@ const createSuccessfulClaimsMapping = (eligibilityStatus) => {
   })
 }
 
+const createUpdatedClaimsMapping = () => {
+  return JSON.stringify({
+    'request': {
+      'method': 'POST',
+      'url': '/v2/claims',
+      'headers': {
+        'X-Request-ID': {
+          'matches': ID_HEADERS_MATCH
+        },
+        'X-Session-ID': {
+          'matches': ID_HEADERS_MATCH
+        }
+      }
+    },
+    'response': {
+      'status': 200,
+      'jsonBody': {
+        'claimStatus': 'ACTIVE',
+        'eligibilityStatus': 'ELIGIBLE',
+        'claimUpdated': true,
+        'updatedFields': updatedFields,
+        voucherEntitlement
+      },
+      'headers': {
+        'Content-Type': 'application/json'
+      }
+    }
+  })
+}
+
 const ERROR_CLAIMS_MAPPING = JSON.stringify({
   'request': {
     'method': 'POST',
-    'url': '/v1/claims',
+    'url': '/v2/claims',
```

Comment:
Would it be worthwhile having this as a constant seeing as you had up date it 3 times?

Reply:
I see there is already one, so let's use it.

Reply:
Added in the test constants file and using that.